core/src/lib.rs

```diff
@@ -15,6 +15,7 @@ pub mod stream;
 pub mod sumeragi;
 pub mod torii;
 pub mod tx;
+mod wasm;
```

Comment:
As `wasm` execution is an alternative to `isi` direct execution. We might want to have these modules at the same level. I think we should either remove `smartcontracts` folder from hierarchy, so we would move down the `isi`. Or we should move `wasm` into `smartcontracts`.

Reply:
I'll move wasm into `smartcontract`.